squad/frontend/comparison.py

```diff
@@ -39,6 +39,8 @@ def __get_transitions(request):
 
     if len(marked_transitions) > 0:
         for t in marked_transitions:
+            if t is None:
```

Comment:
It's serialized, so should be sth like: 
```
if t is None or t == "None":
```
Right now it throws exception.

Reply:
nice spot, just pushed a fix